docs/howto/advanced_filtering/config.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+core:
+    num_workers: 20
 inventory:
     plugin: nornir.plugins.inventory.simple.SimpleInventory
     options:
```

Comment:
can we add here `user_defined` for completeness?

Reply:
I think we should add a proper example about that instead of in here, which is for something else so let's treat this as a quick and dirty documentation hack to avoid hogging this even more